serverless/knative_serving/serverless-applications.adoc

```diff
@@ -34,3 +34,11 @@ include::modules/kn-service-describe.adoc[leveloffset=+1]
 
 include::modules/verifying-serverless-app-deployment.adoc[leveloffset=+1]
 include::modules/interacting-serverless-apps-http2-gRPC.adoc[leveloffset=+1]
+
+== Using kn CLI in offline mode
+
+:FeatureName: The offline mode of the kn CLI
```

Comment:
```suggestion
:FeatureName: The offline mode of the `kn` CLI
```

Should this be `kn`?

Reply:
@abrennan89 No, it won't render correctly in the included Tech Preview note.